proto/stride/ratelimit/ratelimit.proto

```diff
@@ -49,3 +49,8 @@ message RateLimit {
   Quota quota = 2;
   Flow flow = 3;
 }
+
+message WhitelistedAddressPair {
```

Comment:
My vote is option 1, I don't see any reason to add a restriction here since either a) the other chains don't have a stride address and a transfer would not be possible or b) they do have a stride address, in which case having a restriction here would break things

Reply:
Sgtm - should we just add a comment above the data structure linking to this discussion?